decompy/filtercfiles/filter_c_file.py

```diff
@@ -0,0 +1,197 @@
+import os
```

Comment:
Should filtering C files really be seperate of data gathering like this? I consider it part of making sure to reject invalid data.....